src/Algolia.Search/Clients/AnalyticsClient.cs

```diff
@@ -46,8 +46,19 @@ public class AnalyticsClient : IAnalyticsClient
         /// </summary>
         /// <param name="applicationId">Your application ID</param>
         /// <param name="apiKey">Your Api KEY</param>
-        public AnalyticsClient(string applicationId, string apiKey) : this(
-            new AnalyticsConfig(applicationId, apiKey), new AlgoliaHttpRequester())
+        [Obsolete("This constructor is deprecated, please use AnalyticsClient(string applicationId, string apiKey, string region) instead.")]
+        public AnalyticsClient(string applicationId, string apiKey) : this(applicationId, apiKey, "us")
```

Comment:
I think we can leverage optional parameter to avoid Deprecate method. Something like this should work:

```csharp
public AnalyticsClient(string applicationId, string apiKey, string region = "us")
```

Reply:
Oh right, I forgot you had this in C# (too much context switching these days).